src/SprykerEco/Zed/AmazonQuicksight/Business/AmazonQuicksightFacadeInterface.php

```diff
@@ -52,36 +52,25 @@ public function createQuicksightUsersForUserCollectionResponse(
 
     /**
      * Specification:
-     * - Requires `AnalyticsEmbedUrlRequestTransfer.user` and `AnalyticsEmbedUrlRequestTransfer.user.idUser` to be set.
-     * - Returns `true` if Quicksight user with the provided user ID exists in DB.
-     * - Returns `false` otherwise.
+     * - Requires `AnalyticsRequestTransfer.user` to be set.
+     * - Requires `AnalyticsRequestTransfer.user.quicksightUser` to be set.
+     * - Requires `AnalyticsRequestTransfer.user.quicksightUser.arn` to be set.
```

Comment:
Not sure if we need this since we still fetch quicksight user from the database. 
We can use fetched transfer and not rely on the provided one.